src/MessageBird/Resources/Conversation/Messages.php

```diff
@@ -123,6 +124,25 @@ public function getList($conversationId, $parameters = array())
         return $this->processRequest($body);
     }
 
+    public function readMessage($messageId, $parameters = array()) {
+        list($status, , $body) = $this->httpClient->performHttpRequest(
+            HttpClient::REQUEST_GET,
+            sprintf(self::MESSAGE_RESOURCE_NAME, $messageId),
+            $parameters
+        );
+
+        if ($status === self::HTTP_STATUS_OK) {
+            $body = json_decode($body);
```

Comment:
If the body fails to decode as JSON, then `$body` will be `null`. Could we add some error handling here?